proto/frequenz/dispatch/v1/dispatch.proto

```diff
@@ -1,6 +1,32 @@
-// Frequenz Dispatch API
+// Frequenz Dispatch Automation API
 //
-// Frequenz gRPC API to propagate dispatches to microgrids
+// Overview:
+// The Frequenz dispatch gRPC API serves to automate the process of electricity dispatches for microgrids.
```

Comment:
This was slightly tweaked in the issue https://github.com/frequenz-floss/frequenz-api-dispatch/issues/60